build/pause/Makefile

```diff
@@ -69,7 +69,7 @@ TRIPLE.linux-s390x := s390x-linux-gnu
 TRIPLE := ${TRIPLE.${OS}-${ARCH}}
 BASE.linux := scratch
 # Source for windows pause image base is located at https://github.com/microsoft/windows-pause-image-base
-BASE.windows := mcr.microsoft.com/oss/kubernetes/windows-pause-image-base:v0.4.1@sha256:37cc10768383b55611d724a05eb18564cb5184c89b0c2faa7d4eff63475092df
+BASE.windows := mcr.microsoft.com/oss/kubernetes/windows-pause-image-base:v0.4.1
```

Comment:
if there's not any good reason to rush, I think we should fix this to stop mutating the tag in other parts of the build

perhaps we need multiple base image variables for windows, and _each of them_ should be @digest

Reply:
using tag is less secure and not reproducible

Reply:
>perhaps we need multiple base image variables for windows, and each of them should be @digest

As @BenTheElder mentinoed, this looks to be the correct way of solving this. Can you check if we can fix it @ArkaSaha30 ?

Reply:
> using tag is less secure and not reproducible

Yes, agree. I will update this PR with the fix.
Thank you